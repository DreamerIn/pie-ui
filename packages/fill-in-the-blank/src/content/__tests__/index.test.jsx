import { shallow } from 'enzyme';
import React from 'react';

import { Content } from '../index';

describe('Content', () => {
  let w;
  let onChange = jest.fn();
  const wrapper = extras => {
    const defaults = {
      classes: {
        content: 'content',
        holder: 'holder'
      },
      className: 'className',
      onChange,
      content: [
        'test',
        {
          id: '1',
          choice: {
            id: '1',
            content: 'foo'
          }
        }
      ]
    };
    const props = { ...defaults, ...extras };
    return shallow(<Content {...props} />);
  };
  describe('snapshot', () => {
    it('renders', () => {
      w = wrapper();
      expect(w).toMatchSnapshot();
    });
  });
  describe('logic', () => {});
});
