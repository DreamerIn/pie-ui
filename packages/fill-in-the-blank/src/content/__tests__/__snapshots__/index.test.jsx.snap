// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content snapshot renders 1`] = `
<div
  className="content className"
>
  <div
    className="holder"
  >
    <Entry
      content="test"
      key="0"
      onDropChoice={[Function]}
      onRemoveChoice={[Function]}
    />
    <Entry
      content={
        Object {
          "choice": Object {
            "content": "foo",
            "id": "1",
          },
          "id": "1",
        }
      }
      key="1"
      onDropChoice={[Function]}
      onRemoveChoice={[Function]}
    />
  </div>
</div>
`;
